fsevents.js

```diff
@@ -6,6 +6,13 @@
 /* jshint node:true */
 'use strict';
 
+if (process.platform !== 'darwin') {
+  var err = new Error('Cannot find module \'fsevents\'');
```

Comment:
My eyes are going bad. Misread the latest docs for `spawn's` `option.stdio`, where it stated "Added in v0.7.10"; missed the first "0" there, and read as "v7.10".